src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -7,54 +7,61 @@
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
     "family" : "4",
-    "tags" : [ "buyer" ]
+    "tags" : [ "buyer" ],
+    "remark" : "this is a remark"
   }, {
     "name" : "Benson Meier",
     "phone" : "98765432",
     "income" : "20000",
     "email" : "johnd@example.com",
     "address" : "311, Clementi Ave 2, #02-25",
     "family" : "4",
-    "tags" : [ "seller", "buyer" ]
+    "tags" : [ "seller", "buyer" ],
+    "remark" : ""
   }, {
     "name" : "Carl Kurz",
     "phone" : "95352563",
     "income" : "30000",
     "email" : "heinz@example.com",
     "address" : "wall street",
     "family" : "4",
-    "tags" : ["buyer"]
+    "tags" : ["buyer"],
+    "remark" : "Carl was supposed to start with a K, but the doctor misspelled when he was born"
   }, {
     "name" : "Daniel Meier",
     "phone" : "87652533",
     "income" : "40000",
     "email" : "cornelia@example.com",
     "address" : "10th street",
     "family" : "4",
-    "tags" : [ "buyer" ]
+    "tags" : [ "buyer" ],
+    "remark" : "White VANS"
   }, {
     "name" : "Elle Meyer",
     "phone" : "9482224",
     "income" : "50000",
     "email" : "werner@example.com",
     "address" : "michegan ave",
     "family" : "4",
-    "tags" : [ "buyer"]
+    "tags" : [ "buyer"],
+    "remark" : ""
   }, {
     "name" : "Fiona Kunz",
     "phone" : "9482427",
     "income" : "60000",
     "email" : "lydia@example.com",
     "address" : "little tokyo",
     "family" : "4",
-    "tags" : ["seller" ]
+    "tags" : ["seller" ],
+    "remark" : ""
   }, {
     "name" : "George Best",
     "phone" : "9482442",
     "income" : "70000",
     "email" : "anna@example.com",
     "address" : "4th street",
     "family" : "4",
-    "tags" : ["buyer" ]
+    "tags" : ["buyer" ],
+    "remark" : "Fun fact: George's brother's name is \"The\"."
```

Comment:
I like this test as it helps as to ensure there is no problem in parsing remarks with symbols